r/R/dplyr-funcs-datetime.R

```diff
@@ -571,11 +571,20 @@ register_bindings_datetime_parsers <- function() {
     }
   })
 
-  ymd_parser_vec <- c("ymd", "ydm", "mdy", "myd", "dmy", "dym", "ym", "my", "yq")
+  parser_vec <- c(
+    "ymd", "ydm", "mdy", "myd", "dmy", "dym", "ym", "my", "yq",
+    "ymd_HMS", "ymd_HM", "ymd_H", "dmy_HMS", "dmy_HM", "dmy_H",
+    "mdy_HMS", "mdy_HM", "mdy_H", "ydm_HMS", "ydm_HM", "ydm_H"
+  )
 
-  ymd_parser_map_factory <- function(order) {
+  parser_map_factory <- function(order) {
     force(order)
     function(x, tz = NULL) {
```

Comment:
`lubridate::parse_date_time()` has a `truncated` argument (which we also support with the `parse_date_time` binding). What do you think about exposing it to the `ymd_...()` family of functions? The same goes for `quiet` where the binding has a different default (is not chatty, `quiet = TRUE`).

Reply:
Added `quiet `, `locale` and `truncated`. However `quiet` and `locale` currently just error if changed from default. I'll open a Jira to address this later.